Grammar/python.gram

```diff
@@ -447,13 +454,19 @@ args[expr_ty]:
                      : CHECK(singleton_seq(p, a)),
                  (b) ? ((expr_ty) b)->v.Call.keywords : NULL,
                  EXTRA) }
-kwargs[asdl_seq*]: a=','.kwarg+ { a }
+kwargs[asdl_seq*]:
+    | a=','.kwarg+ & ')' { a }
```

Comment:
Is this lookahead needed?

Reply:
As far as I understand yes because if not there it can consume partially the argument sequence and won't try the next option

Reply:
For instance, try `f(*args, x=3, **kwargs)` without that lookahead